### PR TITLE
Deleted down websites part 2

### PR DESCRIPTION
**Pull request by toprak**
gg.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,21 +80,16 @@ A curated list of anime streaming/downloading websites.
...
-* [mangahere.co](https://www.mangahere.co/)
+* [mangahere.cc](https://mangahere.cc/)
 * [mangakakalot.com](http://mangakakalot.com/)
-* [manganel.com](http://manganel.com)
+* [manganelo.com](https://manganelo.com//)
 * [mangapanda.com](http://www.mangapanda.com/)
-* [mangapark.me](http://mangapark.me/)
+* [mangapark.net](http://mangapark.net/)
 * [mangareader.net](http://www.mangareader.net/)
-* [mangaseeonline.us](http://mangaseeonline.us/)
```
Comment (anshumanv) [inline README.md:107]: chnage to https://mangasee123.com/

File: README.md [hunk 2]
```diff
@@ -95,21 +80,16 @@ A curated list of anime streaming/downloading websites.
...
-* [mangastream.com](http://mangastream.com/)
 * [ReadManga Today](https://www.readmng.com/)
-* [zingbox.me](http://www.zingbox.me/)
 
 ### MANGA (mobile app)
 * [mangazone](https://play.google.com/store/apps/details?id=com.ng.mangazone&hl=en)
-* [Manga Rock](https://play.google.com/store/apps/details?id=com.notabasement.mangarock.android.titan)
```
Comment (anshumanv) [inline README.md:114]: change to https://mangarock.com/getapp

**Review by anshumanv — CHANGES_REQUESTED** [review]
Some comments, Can you also add a section for Anime apps and add - https://zunjae.github.io/anymeapp.com/ and others?

File: README.md [hunk 3]
```diff
@@ -95,21 +80,16 @@ A curated list of anime streaming/downloading websites.
...
 * [bulumanga.com](http://bulumanga.com/)
-* [jaiminisbox.com](https://jaiminisbox.com/)
-* [mangafox.me](https://mangafox.me/)
+* [fanfox.net](https://fanfox.net/)
 * [mangafreak.eu](http://mangafreak.eu/)
-* [mangahere.co](https://www.mangahere.co/)
+* [mangahere.cc](https://mangahere.cc/)
 * [mangakakalot.com](http://mangakakalot.com/)
-* [manganel.com](http://manganel.com)
+* [manganelo.com](https://manganelo.com//)
```
Comment (anshumanv) [inline README.md:87]:
```suggestion
* [manganelo.com](https://manganelo.com/)
```

File: README.md (hunk 2 above)
Comment (toprak) [reply README.md:114]: Mangarock is down so its useless.

File: README.md (hunk 1 above)
Comment (toprak) [reply README.md:107]: I have to go somewhere, I will do big implementation on this, so just wait :)

Comment (toprak) [reply README.md:107]: so I will just change mangasee123.com nowness.

File: README.md (hunk 2 above)
Comment (anshumanv) [reply README.md:114]: down? Site seems to be running, did they announce that it's down?

File: README.md (hunk 1 above)
Comment (anshumanv) [reply README.md:107]: feel free to change when convenient :)

File: README.md (hunk 2 above)
Comment (toprak) [reply README.md:114]: yes ```Hi all

As we announced previously, we have removed all reading features from the website.

Now when you visit each title’s info page, you can no longer read scanlations. Instead, we have added a feature that links each title in Manga Rock to an official content provider’s site (like Comixology, or viz.com etc.) where you can support the original authors and their work.

Disclaimer:  We do not receive any referral fee from any of your purchases.

In the meantime, our team is still working hard on project “MR Comics” where we hope to make comics more accessible to everyone — we will share details with you soon.

If you have any questions or need help (membership, account etc.), please contact  support@mangarock.com```

**Review by anshumanv — APPROVED** [review]
Thanks! ✨

**Pull request merged**